feat(third-party): mention neighbor cell measurements (#220)
NordicSemiconductor/asset-tracker-cloud-docs: docs/app/CellGeolocation.rst

It can be useful, for example when tracking assets like parcels.
The approximate location of a parcel can be combined with known points of interest (for example, warehouses and ports) and you can find out if a shipment has arrived at the destination.

> Precision of cell geolocation can be improved by using neighbor cell measurement reports which were introduced in version 1.3.0 of the nRF9160 modem firmware.

Reviewer: ```suggestion
Precision of cell geolocation can be improved by using neighbor cell measurement reports that were introduced in version 1.3.0 of the nRF9160 modem firmware.
```

> With very little energy costs, the modem can collect a report of all cells in the mobile phone network it currently can see.

Reviewer: ```suggestion
With very little power cost, the modem collects a report of all cells it currently sees in the mobile phone network.
```

> This additional information can be used to provide location estimation with improved accuracy.

Reviewer: ```suggestion
This additional information can be used to improve the accuracy of the location estimation.
```

 
If there is no device geolocation for the cell from your own devices, third-party services like `Unwired Labs <https://unwiredlabs.com/>`_ or `RXNetworks <https://rxnetworks.com/location.io#!RT-GNSS>`_ provide a database of cell geolocations.

Third-party location APIs
*************************

nRF Asset Tracker implements the following third-party location APIs:

 - nRF Connect for Cloud Location Services
 - Unwired Labs

> These are commercial third-party solution that provide an API for resolving cell information to geolocation.

Reviewer: ```suggestion
These are commercial third-party solutions that provide an API for resolving cell information to geolocation.
```
